### PR TITLE
Added various games

### LINKED ISSUE #136: Survivor 1982 Commodore 64 video game remake
Currently no Wikipedia article for 1982 version, therefore can't add

https://github.com/scottschiller/SURVIVOR

2015-04-05 14_16_09-survivor - wikipedia the free encyclopedia

### PR DESCRIPTION
**Pull request by wonea**
Fixes #136 & #137 & #144

### CODE REVIEW

File: games.yaml [hunk 1]
```diff
@@ -2786,6 +2855,14 @@
           image: http://www.oolite.org/images/gallery/coriolis!jpg
         - url: http://www.oolite.org/images/gallery/full/advanced-nav-array
           image: http://www.oolite.org/images/gallery/advanced-nav-array
+  clones:
+    - name: Mission ELITE: The Battle For Birera
```
Comment (piranha) [inline games.yaml]: You have to put quote marks around string if it contains a colon, so should be `- name: "Mission ELITE: The Battle For Birera"`

**Comment by wonea** [comment]
I think its not building because some nodes have both; clones & reimplementations

File: games.yaml [hunk 2]
```diff
@@ -1839,6 +1940,21 @@
         - image: http://freesiege.sourceforge.net/screens/screen0.png
         - image: http://freesiege.sourceforge.net/screens/screen1.png
 
+- name: Siege of Avalon
+  clones:
+    name: Siege of Avalon Open Source
```
Comment (piranha) [inline games.yaml]: You forgot to prepend name with `-` here. :) Unfortunately YAML is a bit brittle. I tried to improve error reporting a bit.

**Comment by piranha** [comment]
Now Travis report is much better.

File: games.yaml (hunk 2 above)
Comment (wonea) [inline games.yaml]: Thanks for your help. Start going cross eyed after a while!

**Comment by piranha** [comment]
Cool!

I wonder if I need to rework this in a static typed language just so that error reporting is better. :))

**Comment by wonea** [comment]
More reporting on build errors is always helpful. Lines number usually are enough, but sometimes the obvious is not obvious!

**Pull request merged**